__tests__/app.test.js

```diff
@@ -45,3 +45,42 @@ describe("200 GET /api", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("200: should respond with 200 status code, and a singular article from the articles table in my database, matching the ID passed into the parameters.", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        const article = body.article;
+        const articleSchema = {
+          article_id: "number",
+          title: "string",
+          topic: "string",
+          author: "string",
+          body: "string",
+          created_at: "string",
+          article_img_url: "string",
+        };
+        for (const key in articleSchema) {
+          expect(articleSchema[key]).toBe(typeof article[key]);
+        }
```

Comment:
This test is valid but as we are dealing with just one object, we can be really specific in our test and use the actual values rather than just the data types. I would suggest changing this at some point 🙂